docs/zh-CN/sql-reference/sql-statements/Data Manipulation/BROKER LOAD.md

```diff
@@ -477,6 +485,21 @@ under the License.
          ORDER BY source_sequence
         ) 
         with BROKER "hdfs" ("username"="user", "password"="pass");
+
+    15. 从 Aliyun OSS 导入csv格式的数据
+        LOAD LABEL example_db.label15
+        (
+        DATA INFILE("oss://my_bucket/input/file.csv")
+        INTO TABLE `my_table`
+        (k1, k2, k3)
+        )
+        WITH BROKER my_oss_broker
+        (
+        "fs.oss.accessKeyId" = "xxxxxxxxxxxxxxxxxxxxxxxxxx",
```

Comment:
```suggestion
        "fs.oss.accessKey" = "xxxxxxxxxxxxxxxxxxxxxxxxxx",
```

Reply:
we use hadoop oss api to access aliyun oss, and the config params is fs.oss.accessKeyId, fs.oss.accessKeySecret. https://developer.aliyun.com/article/292792
And like s3 config, we'd better use the same param to hadoop ecology.